NEWS

```diff
@@ -7,6 +7,10 @@ Version 4.2.0
 * Implement yescrypt ($y$) and scrypt ($7$) hashing algorithms.
 * For scrypt the implemented gensalt function ensures every new hash
   is computed using at least 32 MiBytes of RAM.
+* yescrypt is the new default hashing algorithm used for new hashes.
```

Comment:
This repeats itself.  "yescrypt is now the default algorithm used for new hashes" would be better.

Reply:
Also, s/algorithm/method/ throughout, for consistency.